ci/jenkins/jobs/macros.yaml

```diff
@@ -301,3 +301,47 @@
            #!/bin/bash
            set -e
            ./ci/jenkins/stop-stale-jobs.sh --pull-request "${ghprbPullId}" --jenkins "${JENKINS_URL}"
+
+- builder:
+    name: builder-capa-matrix
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          export JOB_NAME="matrix-${TEST_OS}-k8s-${K8S_VERSION//./-}-build-num"
```

Comment:
I think Jenkins has a built-in variable `JOB_NAME`. We assign a custom value here. Is it because the custom values is more readable? In this case, maybe use `CLUSTER_NAME` as variable name. Same for JOB_NAME in other builders.

Reply:
we can do that as well, i preferred this because it was already in use, when i started working on capa task.